scss/arizona-bootstrap.scss

```diff
@@ -9,6 +9,7 @@
  */
 @import "../node_modules/bootstrap/scss/functions";
 @import "../node_modules/bootstrap/scss/variables";
+@import "variable-overrides";
```

Comment:
The previous suggestion was:
The `scss/_variable-overrides.scss` file isn't used by anything apart from this breadcrumb override, so we can delete it.
So it shouldn't be referenced, and the file itself deleted.